agent/bench-scripts/test-bin/mock-postprocess/dbench-postprocess

```diff
@@ -1,28 +1,29 @@
 #!/usr/bin/env python3
 
-import os, sys
+import os
+import sys
 
 outfn = os.environ.get("_testlog")
 if not outfn:
     sys.stderr.write("WARNING _testlog environment variable not present\n")
     sys.exit(1)
 else:
     with open(outfn, "a") as outf:
-        outf.write(' '.join(sys.argv) + '\n')
+        outf.write(" ".join(sys.argv) + "\n")
 
-#echo "dir=$1"
-#echo "iteration=$2"
-#echo "toollabelprefix=$3"
-#echo "toolgroup=$4"
-#"${count}-${protocol}_${test_type}-${message_size}B-${instance}i"
+# echo "dir=$1"
+# echo "iteration=$2"
+# echo "toollabelprefix=$3"
+# echo "toolgroup=$4"
+# "${count}-${protocol}_${test_type}-${message_size}B-${instance}i"
```

Comment:
I know you're just using `black` to reformat, but it's awfully tempting to ask if we really want to retain these commented-out statements.

I'm struggling to resist. 😆